sources/packages/backend/apps/api/src/route-controllers/application/application.system-access.controller.ts

```diff
@@ -88,41 +84,6 @@ export class ApplicationSystemAccessController extends BaseController {
     }
   }
 
-  /**
```

Comment:
Eventually this whole file is going to be deleted right?

Reply:
Right, all system actions from API will be moved to workers, as far as we know right now.